Sources/JExtractSwiftLib/JNI/JNISwift2JavaGenerator+SwiftThunkPrinting.swift

```diff
@@ -192,19 +202,32 @@ extension JNISwift2JavaGenerator {
     let translatedDecl = self.translatedDecl(for: decl)
     let swiftReturnType = decl.functionSignature.result.type
 
-    let downcallParameters = renderDowncallArguments(
-      swiftFunctionSignature: decl.functionSignature,
-      translatedFunctionSignature: translatedDecl.translatedFunctionSignature
-    )
     let tryClause: String = decl.isThrowing ? "try " : ""
-    let functionDowncall = "\(tryClause)\(calleeName).\(decl.name)(\(downcallParameters))"
+
+    let result: String
+    switch decl.apiKind {
+    case .function, .initializer:
+      let downcallParameters = renderDowncallArguments(
+        swiftFunctionSignature: decl.functionSignature,
+        translatedFunctionSignature: translatedDecl.translatedFunctionSignature
+      )
+      result = "\(tryClause)\(calleeName).\(decl.name)(\(downcallParameters))"
+
+    case .getter:
+      result = "\(tryClause)\(calleeName).\(decl.name)"
+
+    case .setter:
+      assert(decl.functionSignature.parameters.count == 1)
+      let newValueParameter = decl.functionSignature.parameters[0]
```

Comment:
Generally avoid indexing with integers, maybe rephrase this in a `guard let … = ….parameters.first else { … crash here }`